lib/repetitive-task.js

```diff
@@ -52,20 +53,26 @@ class RepetitiveTask {
   }
 
   __process () {
+    let processStartTime
     this.isProcessing = this._getTask()
       .then((task) => {
         if (!task) {
           this.logger.info('No pending tasks')
           return
         }
-
+        this.currentTask = task
         this.logger.info(this._getLogData(task), 'processing task')
+        processStartTime = process.hrtime()
         return this._process(task)
       })
       .catch((error) => {
         this.logger.error({ stack: error.stack }, 'Error processing task')
       })
       .then(() => {
+        const processElapsedTime = { elapsed_time: hrtimeToMs(process.hrtime(processStartTime)) }
```

Comment:
The key in the object is the only thing I care about. I don't think we should be double logging the task data, so there won't be an intermediary object anyway.